modules/remotebackend/testrunner.sh

```diff
@@ -209,9 +221,9 @@ function stop_unix() {
 
 function run_test() {
  if [ $new_api -eq 0 ]; then
-   ./"$mode_name"
+   ./"$mode_name" || rv=$? && rv=0
  else
-    $mode
+   $mode || rv=$? && rv=0
```

Comment:
I'm almost certain this will always set rv to 0. Maybe just leave out the `&& rv=0` bit?